packages/block-library/src/legacy-widget/edit/dom-manager.js

```diff
@@ -67,22 +64,9 @@ class LegacyWidgetEditDomManager extends Component {
 						className="form"
 						ref={ this.formRef }
 						method="post"
-						onBlur={ () => {
-							if ( this.shouldTriggerInstanceUpdate() ) {
-								if ( isReferenceWidget ) {
```

Comment:
This code was autosaving the legacy widgets on each change so I removed it - there is a save button on the widget management screen so autosaving on change is at least unexpected.

Reply:
The non-class based widgets render their own form and perform their own save sometimes using tables they created. This data is outside the structure we use to control the widgets and I guess we need a specific save for it, it can be automatically or we can have a save button on the legacy widgets block.

Reply:
Right now the non-class based widgets seem to not be saved.

I tested with this simple widget https://themes.trac.wordpress.org/browser/silesia/1.0.6/include/widgets/feedburner.php
(pasted it on functions.php)


And I verified the values on the form never get saved.